doc/source/release-notes/Percona-Server-5.7.21-21.rst

```diff
@@ -0,0 +1,135 @@
+.. rn:: 5.7.21-21
+
+========================
+Percona Server 5.7.21-21
+========================
+
+Percona is glad to announce the release of Percona Server 5.7.21-21
+on April 23, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.21-21/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.21
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-21.html>`_
+and includes all the bug fixes in it.
+Percona Server 5.7.21-21 is now the current GA (Generally Available) release
+in the 5.7 series. 
+
+This version of |Percona Server| includes three new encryption features: `Vault
```

Comment:
This release makes the previously-developed encryption features GA, and introduces new beta features

@tduzan